language/tools/move-cli/src/main.rs

```diff
@@ -2,339 +2,13 @@
-    let mode = Mode::new(move_args.mode);
-    match sandbox_command {
-        SandboxCommand::Link { no_republish } => {
-            let state = mode.prepare_state(&move_args.build_dir, &move_args.storage_dir)?;
-            base::commands::check(
-                &[state.interface_files_dir()?],
-                !*no_republish,
-                &[DEFAULT_SOURCE_DIR.to_string()],
-                move_args.verbose,
-            )
-        }
-        SandboxCommand::Publish {
-            source_files,
-            no_republish,
-            ignore_breaking_changes,
-        } => {
-            let state = mode.prepare_state(&move_args.build_dir, &move_args.storage_dir)?;
-            sandbox::commands::publish(
-                &state,
-                source_files,
-                !*no_republish,
-                *ignore_breaking_changes,
-                move_args.verbose,
-            )
-        }
-        SandboxCommand::Run {
-            script_file,
-            script_name,
-            signers,
-            args,
-            type_args,
-            gas_budget,
-            dry_run,
-        } => {
-            let state = mode.prepare_state(&move_args.build_dir, &move_args.storage_dir)?;
-            sandbox::commands::run(
-                &state,
-                script_file,
-                script_name,
-                signers,
-                args,
-                type_args.to_vec(),
-                *gas_budget,
-                *dry_run,
-                move_args.verbose,
-            )
-        }
-        SandboxCommand::Test {
-            path,
-            track_cov: _,
-            create: true,
-        } => sandbox::commands::create_test_scaffold(path),
-        SandboxCommand::Test {
-            path,
-            track_cov,
-            create: false,
-        } => sandbox::commands::run_all(
-            path,
-            &std::env::current_exe()?.to_string_lossy(),
-            *track_cov,
-        ),
-        SandboxCommand::View { file } => {
-            let state = mode.prepare_state(&move_args.build_dir, &move_args.storage_dir)?;
-            sandbox::commands::view(&state, file)
-        }
-        SandboxCommand::Clean {} => {
-            // delete storage
-            let storage_dir = Path::new(&move_args.storage_dir);
-            if storage_dir.exists() {
-                fs::remove_dir_all(&storage_dir)?;
-            }
-
-            // delete build
-            let build_dir = Path::new(&move_args.build_dir);
-            if build_dir.exists() {
-                fs::remove_dir_all(&build_dir)?;
-            }
-            Ok(())
-        }
-        SandboxCommand::Doctor {} => {
-            let state = mode.prepare_state(&move_args.build_dir, &move_args.storage_dir)?;
-            sandbox::commands::doctor(&state)
-        }
-    }
+    let error_descriptions: ErrorMapping =
+        bcs::from_bytes(move_stdlib::error_descriptions()).unwrap();
```

Comment:
Since `main` returns `Result`, should we replace this `unwrap`+ the one below with `?`'s?